person.rb

```diff
@@ -9,9 +9,14 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] As the reviewer addressed before, please pass the person as a parameter to a new rental object. So you need to create a rental here. Pass (book, date, self) as an argument to create a rental. And add that rental to person's rentals with `@rentals << rental`